README.md

```diff
@@ -253,7 +253,17 @@ Supported sections are `[ats]`, `[atsmp]`, `[ets2]` and `[ets2mp]`.
 
 Key-Value-Pair|Description
 ---|---
-`without-rich-presence = [boolean]`|Disable Discord Rich Presence for the game mode - overriding what is set in third party sections.
+`without-rich-presence = [boolean]`|Disable Discord Rich Presence for the game mode - overriding what is set in third party sections. [Default: `no`]
+`game-directory = [path]`|Choose a different directory for the game files [Default: `$XDG_DATA_HOME/truckersmp-cli/(Game name)/data`]
+`prefix-directory = [path]`|Choose a different directory for the prefix [Default: `$XDG_DATA_HOME/truckersmp-cli/(Game name)/prefix`]
+`game-options = [options]`|Specify ATS/ETS2 options Note: If specifying one option, use `--game-options=-option` format [Default: `-nointro -64bit`]
```

Comment:
I'm not entirely sure what you're trying to say here.

Should the config entry look like `game-options = --game-options=-option` or **has** this to be a parameter in the program call `truckersmp-cli --game-options=-option`?

```suggestion
`game-options = [options]`|Specify ATS/ETS2 options Note: If specifying only one option, use `--game-options=-option` format [Default: `-nointro -64bit`]
```